### PR TITLE
[Config] Rename argument`save_model_per_iters` to `save_model_frequency`

### PR DESCRIPTION
**Pull request by isratnisa**
Resolves [Issue #116 ](https://github.com/awslabs/graphstorm/issues/116)

### CODE REVIEW

File: python/graphstorm/config/argument.py [hunk 1]
```diff
@@ -541,34 +541,34 @@ def save_model_path(self):
         return None
 
     @property
-    def save_model_per_iters(self):
```
Comment (classicsong) [inline python/graphstorm/config/argument.py:544]: You also need to update --save-model-per-iters

Comment (classicsong) [reply python/graphstorm/config/argument.py:544]: Also also check if any test scripts use --save-model-per-iters

Comment (isratnisa) [reply python/graphstorm/config/argument.py:544]: Not following. I did replace `save_model_per_iters` with `save_model_frequency`.

File: tests/end2end-tests/graphstorm-lp/mgpu_test.sh [hunk 2]
```diff
@@ -208,7 +208,8 @@ then
 fi
 
 echo "**************dataset: Movielens, RGCN layer 2, node feat: fixed HF BERT & sparse embed, BERT nodes: movie, inference: full-graph, negative_sampler: joint, exclude_training_targets: true, save model, early stop"
-python3 $DGL_HOME/tools/launch.py --workspace $GS_HOME/training_scripts/gsgnn_lp --num_trainers $NUM_TRAINERS --num_servers 1 --num_samplers 0 --part_config /data/movielen_100k_lp_train_val_1p_4t/movie-lens-100k.json --ip_config ip_list.txt --ssh_port 2222 "python3 gsgnn_lp.py --cf ml_lp.yaml --fanout '10,15' --n-layers 2 --mini-batch-infer false  --use-node-embeddings true --exclude-training-targets True --reverse-edge-types-map user,rating,rating-rev,movie --num-gpus $NUM_TRAINERS --part-config /data/movielen_100k_lp_train_val_1p_4t/movie-lens-100k.json --save-model-path /data/gsgnn_lp_ml_dot/ --topk-model-to-save 3 --save-model-per-iter 1000 --save-embed-path /data/gsgnn_lp_ml_dot/emb/ --enable-early-stop True --early-stop-burnin-rounds 3 -e 30 --window-for-early-stop 2 --early-stop-strategy consecutive_increase" | tee exec.log
+
```
Comment (classicsong) [inline tests/end2end-tests/graphstorm-lp/mgpu_test.sh:211]: Can you remove this empty line?

**Review by classicsong — APPROVED** [review]
On comment.
Overall LGTM.
Please resolve the merge conflicts.

**Pull request merged**